libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -6,6 +6,7 @@
       type="search"
       placeholder="Search for books to add to your reading list"
       formControlName="term"
+      (ngModelChange)="throttleSearchQuery($event)"
```

Comment:
While this kind of works, there is a better way to get the values from the control.

Reply:
updated